saleor/channel/migrations/0006__auto_20221208_0900.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.16 on 2022-12-08 09:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("channel", "0005_channel_allocation_strategy"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="channel",
+            name="automatically_confirm_all_new_orders",
+            field=models.BooleanField(default=True),
+        ),
+        migrations.RunSQL(
```

Comment:
We should add comments with a description that we are adding the database default value to not break backward database compatibility.